app/pathogen/arbovirus/dashboard/(map)/MapShadingLegend.tsx

```diff
@@ -0,0 +1,36 @@
+import { MapSymbology } from "@/app/pathogen/sarscov2/dashboard/(map)/map-config";
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
+import { lastIndexOf } from "lodash";
+
+interface MapShadingLegendProps {
+    className?: string;
+  }
+
+
+export const MapShadingLegend = ({className}: MapShadingLegendProps) => {
+    const shadingLegendElements: {color: string, description: string}[] = [
+        {color: MapSymbology.CountryFeature.HasData.Color, description: "Seroprevalence estimates"},
+        {color: MapSymbology.CountryFeature.Default.Color, description: "No Seroprevalence estimates"}
+    ]
+
+    return (
+        <Card className={className}>
```

Comment:
Similar to above, should localize styling -- Let us preserve using the className prop ONLY for reusable components

Reply:
I localized it!